spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/pulsar/PulsarProperties.java

```diff
@@ -916,6 +930,53 @@ public void setReadCompacted(boolean readCompacted) {
 			this.readCompacted = readCompacted;
 		}
 
+		public Startup getStartup() {
+			return this.startup;
+		}
+
+	}
+
+	public static class Startup {
+
+		/**
+		 * The max time to wait for the container to start.
```

Comment:
Description of configuration properties do not start with `the`, `a`, etc. I suggest `Time to wait for the container to start.`